RELEASE/scripts/autoscend/quests/level_any.ash

```diff
@@ -697,3 +697,120 @@ boolean LX_meatMaid()
 
 	return false;
 }
+
+string LX_getDesiredWorkshed(){
+	string currentWorkshed = get_property("auto_workshed").to_lower_case();
+	//return the actual item name in case a shorthand is used
+	switch(currentWorkshed)
+	{
+		case "model train set":
+		case "train":
+			return "model train set";
+		case "cold medicine cabinet":
+		case "cmc":
+			return "cold medicine cabinet";
+		case "diabolic pizza cube": //unsupported
+		case "pizza": //unsupported
+			return "diabolic pizza cube";
+		case "Asdon Martin keyfob":
+		case "asdon":
+			return "Asdon Martin keyfob";
+		case "portable mayo clinic":
+		case "mayo":
+			return "portable mayo clinic";
+		case "little geneticist dna-splicing lab":
+		case "dnalab":
+			return "little geneticist dna-splicing lab";
+		case "snow machine": //unsupported
+		case "spinning wheel": //unsupported
+		case "warbear auto-anvil": //unsupported
+		case "warbear chemistry lab": //unsupported
+		case "warbear high-efficiency still": //unsupported
+		case "warbear induction oven": //unsupported
+		case "warbear jackhammer drill press": //unsupported
+		case "warbear lp-rom burner": //unsupported
+		default:
+			// auto_workshed is invalid or none/false/whatever to say don't do this

```

Comment:
Could use mafia function `user_confirm()` to ask the user if they would like to continue with the script's automated workshed selector or instead change the setting to a different workshed. Not a big deal either way